src/component-library/NumberInput/NumberInput.tsx

```diff
@@ -50,13 +50,22 @@ const NumberInput = forwardRef<HTMLInputElement, NumberInputProps>(
       return () => input?.removeEventListener('wheel', handleWheel);
     }, [inputRef]);
 
+    const handleKeyDown: KeyboardEventHandler<HTMLInputElement> = (e) => {
```

Comment:
Does not allow user to enter commas. Should this be here or in the TokenField? @tomjeatt

Reply:
Here I would think, assuming the token field extends/uses this component